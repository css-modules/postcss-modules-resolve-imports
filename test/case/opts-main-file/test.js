'use strict';

const resolveImports = require('../../../index');
const setup = require('../../setup');

test('opts-main-file', () => {
  const {resulting, exports: tokens} = setup(
    'local-by-default',
    'extract-imports',
    'scope',
    resolveImports({
      resolve: {
        mainFile: 'a.css',
      },
    })
  )(__dirname);

  expect(resulting).toMatchSnapshot();
  expect(tokens).toMatchSnapshot();
});
