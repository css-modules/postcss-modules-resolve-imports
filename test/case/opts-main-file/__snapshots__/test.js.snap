// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`opts-main-file 1`] = `
"/*  imported from lib/a.css  */

._a_a
{
  color: yellow;
}

/*  imported from source.css  */

._source_b
{
}
"
`;

exports[`opts-main-file 2`] = `
Object {
  "b": "_source_b _a_a",
}
`;
